tera-templates/docsrs/about.html

```diff
@@ -0,0 +1,195 @@
+            </tbody>
+        </table>
+
+        <h4>Badges</h4>
+
+        <p>
+            You can use badges to show state of your documentation to your users.
+            The default badge will be pointed at the latest version of a crate.
+            You can use <code>version</code> parameter to show status of documentation for
+            any version you want.
+        </p>
+
+        <p>
+            Badge will display in blue if docs.rs is successfully hosting your crate
+            documentation, and red if building documentation failing.
+        </p>
+
+        <p>Example badges for mio crate:</p>
+        <table class="pure-table pure-table-horizontal">
+            <thead>
+                <tr>
+                    <th>URL</th>
+                    <th>Badge</th>
+                </tr>
+            </thead>
+
+            <tbody>
+                {%- set mio_badge = "https://docs.rs/mio/badge.svg" -%}
+                <tr>
+                    <td>Latest version: <a href="{{ mio_badge }}">{{ mio_badge }}</a></td>
+                    <td><img src="{{ mio_badge }}" alt="mio" /></td>
+                </tr>
+
+                <tr>
+                    <td>
+                        Version 0.4.4: <a href="{{ mio_badge }}?version=0.4.4">{{ mio_badge }}?version=0.4.4</a>
+                    </td>
+                    <td><img src="{{ mio_badge }}?version=0.4.4" alt="mio" /></td>
+                </tr>
+
+                <tr>
+                    <td>
+                        Version 0.1.0: <a href="{{ mio_badge }}?version=0.1.0">{{ mio_badge }}?version=0.1.0</a>
+                    </td>
+                    <td><img src="{{ mio_badge }}?version=0.1.0" alt="mio" /></td>
+                </tr>
+            </tbody>
+        </table>
+
+        <h4 id="metadata"><a href="#metadata">Metadata for custom builds</a></h4>
+
+        <p>
+            You can customize docs.rs builds by defining <code>[package.metadata.docs.rs]</code>
+            table in your crates' <code>Cargo.toml</code>.
+        </p>
+
+        <p>The available configuration flags you can customize are:</p>
+
+        {# Include the example file #}
+        <pre><code>{%- include "docsrs/Cargo.toml.example" -%}</code></pre>
+
+        <h4>Test crate documentation build locally</h4>
+        {%- set build_subcommand = docsrs_repo ~ "/blob/master/README.md#build-subcommand" -%}
+        <p>
+            The <a href="{{ build_subcommand }}">docs.rs README</a> describes how to build an
+            unpublished crate's documentation locally using the same build nvironment as the build agent.
+        </p>
+
+        <h4>Version</h4>
+        <p>Currently running Docs.rs version is: <strong>{{ docsrs_version() }}</strong></p>
+
+        <h4>Contact</h4>
+        {%- set governance_link = "https://www.rust-lang.org/governance/teams/dev-tools#docs-rs" -%}
+        <p>
+            Docs.rs is run and maintained by the <a href="{{ governance_link }}">Docs.rs team</a>.
```

Comment:
Again, why make a variable that's only used once?

Reply:
For this one it was because it's super duper long